apps/web/src/app/billing/organizations/secrets-manager/sm-adjust-subscription.component.html

```diff
@@ -0,0 +1,92 @@
+<form [formGroup]="formGroup" [bitSubmit]="submit">
+  <bit-form-field class="tw-w-2/3">
+    <bit-label>{{ "subscriptionSeats" | i18n }}</bit-label>
+    <input bitInput id="smSeatCount" formControlName="seatCount" type="number" step="1" min="0" />
```

Comment:
You're using reactive form validation, so you don't need the native html validators:

```suggestion
    <input bitInput id="smSeatCount" formControlName="seatCount" type="number" step="1" />
```

Reply:
The reactive form validation doesn't need the native html validators; but they are required in order to enforce the limit on the native number input increment/decrement controls. Quick demo showing with the native validator and without:

https://github.com/bitwarden/clients/assets/8764515/efe0eeb1-cfd6-4925-b127-ad7d2d0529b9

Reply:
I didn't know this, thanks for the demo! As discussed, please follow up to see if we can put this in our CL somehow.